src/service/performance-impl.js

```diff
@@ -404,8 +404,17 @@ export class Performance {
     }
 
     if (this.supportsNavigation_) {
-      const navigationObserver = this.createPerformanceObserver_(processEntry);
-      navigationObserver.observe({type: 'navigation', buffered: true});
+      // Wrap in a try statement as there are some browsers (ex. chrome 73)
+      // that will say it supports navigation but throws.
+      try {
+        const navigationObserver = this.createPerformanceObserver_(
+          processEntry
+        );
+        navigationObserver.observe({type: 'navigation', buffered: true});
+      } catch (err) {
+        dev() /*OK*/
```

Comment:
should this be an `expectedError`?

Reply:
hmm i was thinking in this case it wouldn't as this shouldn't really be happening